### DISCUSSION TITLE
How do I manage (pdf) bookmarks in Quarto?

### DISCUSSION THREAD

**Discussion by jtlandis** [quarto-dev/quarto-cli#8816] [Q&A]
### Description

Hello, I've been using quarto for a while now and I love its functionality. I have noticed that the default for rendering to pdf is to make bookmarks according to the headings. For example

```{qmd}
---
title: test
format: pdf
---

# Heading 1

## Heading 2

### Heading 3

#### Heading 4

```

would produce a simple pdf, with 3 levels of bookmarks (heading 4 does not produce a bookmark) all collapsed.

Do the authors of quarto know how I might create bookmarks without using a heading? This may look something like the below:

```{qmd}
---
title: test
format: pdf
---

# Heading 1

a)  <!-- foo --> 
b)  <!-- bar -->

## Heading 2

### Heading 3

#### Heading 4

```

Where I could potentially create bookmarks for sub-listings a) and b) either via a shortcode or some class/identifier?

**Comment by mcanouil** [comment]
Bookmark are automatically handled by LaTeX.
You need to check on LaTeX dedicated places if there is a LaTeX command to do that that does not imply the use of `\section` or `\subsection`.
That's not really up to Quarto (or Pandoc).

If you find a command to do that, you'll likely need to use raw code:
- `` `whatever latex code`{=latex} ``
- `` ```{=latex} ``

See also the available options to configure the LaTeX template at <https://quarto.org/docs/reference/formats/pdf.html>.

Edit: the following might be of interest: <https://tex.stackexchange.com/questions/628506/adding-bookmarks-and-subbookmarks-manually-in-regular-latex-file>.

````qmd
---
title: "Quarto Playground"
format: pdf
---

This is a playground for Quarto.

## Markdown

This is a markdown cell.

Fake bookmark `\pdfbookmark[2]{Fake Bookmark}{X}`{=latex}

## Code

Code
````
